art/defences/preprocessor/spatial_smoothing.py

```diff
@@ -132,7 +153,6 @@ def fit(self, x, y=None, **kwargs):
     def set_params(self, **kwargs):
         """
         Take in a dictionary of parameters and applies defence-specific checks before saving them as attributes.
-
```

Comment:
Please keep this empty line because it is important for the correct formatting of the docstring with sphinx.